packages/block-editor/src/components/list-view/block-select-button.js

```diff
@@ -184,6 +186,34 @@ function ListViewBlockSelectButton(
 							</span>
 						</Tooltip>
 					) }
+					{ images.length ? (
+						<span
+							className="block-editor-list-view-block-select-button__images"
+							aria-hidden
+						>
+							{ images
+								.slice( 0, MAX_IMAGES )
+								.map( ( image, index ) => (
+									<span
+										className="block-editor-list-view-block-select-button__image"
+										key={ `img-${ image.url }` }
+										style={ {
+											backgroundImage: `url(${ image.url })`,
+											zIndex: images.length - index, // Ensure the first image is on top, and subsequent images are behind.
+										} }
+									/>
+								) ) }
+							{ images.length > MAX_IMAGES && (
+								<span className="block-editor-list-view-block-select-button__image-count">
+									{ sprintf(
+										/* translators: %d: Number of additional images within a block. */
+										__( '+%d' ),
```

Comment:
This is really just a plus with a number, does it need translation? And if we assume the plus could be translated to something with more than one character, would it fit in the available space?

Reply:
It's mostly about whether or not it's to the left or to the right of the number, rather than being translated to another character. Happy to remove the translation call if we think it's not needed, though!

Reply:
I'd tend towards keeping the translation just in case some locales use a different symbol or order.